eng/pipelines/common/templates/steps/ensure-dotnet-version.yml

```diff
@@ -0,0 +1,77 @@
+#################################################################################
+# Licensed to the .NET Foundation under one or more agreements.                 #
+# The .NET Foundation licenses this file to you under the MIT license.          #
+# See the LICENSE file in the project root for more information.                #
+#################################################################################
+parameters:
+- name: version
```

Comment:
Since we're working on improving our pipelines (and I'm pushing for some degree of uniformity), can we have these with:
* 4 space indents at each level
* Extra line between parameter definitions and tasks
* Sort parameters alphabetically